ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsSet.scala

```diff
@@ -21,7 +21,8 @@ object MetricsSet {
       objectives: Option[TransactionObjectives]
   ): List[Metric[GetTransactionsResponse]] =
     transactionMetrics[GetTransactionsResponse](
-      countingFunction = _.transactions.length,
+      // counting all events across all transactions
+      countingFunction = _.transactions.map(_.events.size).sum,
```

Comment:
how many `_.transactions` there could be?
just wondering how beneficial would be to use `foldLeft` instead of `.map().sum`

Reply:
I don't know, but probably small enough